CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-### CHANGE LOG
+## CHANGE LOG
 
-#### v0.2.0
+### v0.2.0
```

Comment:
Should we bump a version? Since we are using React 16 and Webpack 4. I saw you updated package.json